spec/spec_helper_acceptance.rb

```diff
@@ -40,6 +40,12 @@
       shell 'apt-get install -y ssl-cert'
     end
 
+    if fact('operatingsystem') == 'SLES'
+      #generate certs for SLES
+      shell 'zypper install -y apache2'
```

Comment:
i wasn't sure what to do here. gensslcert is part of apache2 and seemed like the easiest route to uniform certificate creation across SLES versions. there is the whole circumventing the module thing. i could move gensslcert activity into only the tests that require it.